keys/keys_test.go

```diff
@@ -20,6 +20,7 @@ package keys
 import (
```

Comment:
Looks like you made this file executable (mode bits changed from 644 to 755). I assume that was accidental. If not, please justify the change.